src/data.py

```diff
@@ -299,9 +301,25 @@ def fill_empty_menus(eateries):
       base_event = operating_hour.events[0]
       for event in operating_hour.events:
         if not event.menu:
+          # print never shows up so I think this method is now useless
+          print('filling menu for {} on {}'.format(eatery.name, operating_hour.date))
           event.menu = base_event.menu
           event.station_count = base_event.station_count
 
+def merge_hours(eateries):
+  for eatery in eateries.values():
+    for operating_hour in eatery.operating_hours:
+      # need to choose event with data we want to copy
+      if len(operating_hour.events) <= 1:  # ignore these
+        continue
+      base_event = operating_hour.events[0]
+      for event in operating_hour.events[:]:
```

Comment:
`[:]`? this doesn't do anything, did you mean `[1:]`?

Reply:
hm nifty but difficult to read, can you leave a comment?

Reply:
https://unspecified.wordpress.com/2009/02/12/thou-shalt-not-modify-a-list-during-iteration/